### PR TITLE
Px to em

### PR DESCRIPTION
**Pull request by stanislavshylo**

### CODE REVIEW

File: src/_scss/_variables.scss [hunk 1]
```diff
@@ -1,16 +1,20 @@
+@import "functions";
+
+$default-font-size: 16px;
```
Comment (alexkravets) [inline src/_scss/_variables.scss]: What's the point of having this, just use `$font-size` variable.

File: src/_scss/functions.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (alexkravets) [inline src/_scss/functions.scss]: Remove empty lines

Comment (alexkravets) [inline src/_scss/functions.scss]: Follow code convention and rename to `_functions.scss`

File: src/_scss/functions.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+//Convert px to em
```
Comment (alexkravets) [inline src/_scss/functions.scss]: `// Calculate em from px`

**Comment by alexkravets** [comment]
Do not change file formatting, only change values. Keep the source as much unchanged as possible. I mean places like this: http://files.slatestudio.com/ldkS

**Review by alexkravets — CHANGES_REQUESTED** [review]

**Comment by alexkravets** [comment]
Change version to `v6.6.0` — in package.json, gulp if needed and scss

File: src/_scss/_variables.scss [hunk 4]
```diff
@@ -1,16 +1,31 @@
+$font-size: 16px !default;
+
+// Calculate em from px
+@function em($pixels, $context: $font-size) {
```
Comment (alexkravets) [inline src/_scss/_variables.scss]: Rename `$context` to `$base-font-size`

**Review by alexkravets — CHANGES_REQUESTED** [review]

**Comment by stanislavshylo** [comment]
function `em` and `$font-size` should be at top of file `src/_scss/_variables.scss`

**Pull request merged**